database/src/main/java/info/metadude/android/eventfahrplan/database/repositories/SessionsDatabaseRepository.kt

```diff
@@ -29,6 +30,29 @@ class SessionsDatabaseRepository(
         }
     }
 
+    /**
+     * Inserts the session ID into the [SessionByNotificationIdTable] and returns
+     * the newly generated notification ID which is associated with the session ID.
+     *
+     * Notification IDs are incremented automatically.
+     *
+     * [sessionIdContentValues] is expected to be composed from
+     * [SessionByNotificationIdTable.Columns.SESSION_ID] and the session ID value.
```

Comment:
Why not use the session ID as the method parameter?

Reply:
Simply for consistency reasons. All `toConentValues()` conversions happen in the AppRepository class.